The following may be needed on MSVC 8 if lp_rlp.c is regenerated from
a newer Bison
Index: lp_rlp.c
===================================================================
--- lp_rlp.c	(revision 2709)
+++ lp_rlp.c	(working copy)
@@ -1074,7 +1074,6 @@
 #else
 int
 lp_yyparse ()
-    ;
 #endif
 #endif
 {
